reference/docs-conceptual/dsc/resources/authoringResourceComposite.md

```diff
@@ -126,31 +135,46 @@ Configuration xVirtualMachine
 }
 ```
 
+> [!NOTE]
+> DSC doesn't currently support placing composite resources or nested configurations within a
+> composite resource.
+
 ### Saving the configuration as a composite resource
 
-To use the parameterized configuration as a DSC resource, save it in a directory structure like that of any other MOF-based resource, and name it with a **.schema.psm1** extension. For this example, we'll name the file **xVirtualMachine.schema.psm1**. You also need to create a manifest named **xVirtualMachine.psd1** that contains the following line. Note that this is in addition to **MyDscResources.psd1**, the module manifest for all resources under the **MyDscResources** folder.
+To use the parameterized configuration as a DSC resource, save it in a directory structure like that
+of any other MOF-based resource, and name it with a **.schema.psm1** extension. For this example,
+we'll name the file **xVirtualMachine.schema.psm1**. You also need to create a manifest named
+**xVirtualMachine.psd1** that contains the following line.
 
 ```powershell
 RootModule = 'xVirtualMachine.schema.psm1'
 ```
 
+> [!NOTE]
+> This is in addition to **MyDscResources.psd1**, the module manifest for all resources under the
+> **MyDscResources** folder.
+
 When you are done, the folder structure should be as follows.
 
 ```
 $env: psmodulepath
     |- MyDscResources
-           MyDscResources.psd1
+        |- MyDscResources.psd1
         |- DSCResources
             |- xVirtualMachine
                 |- xVirtualMachine.psd1
                 |- xVirtualMachine.schema.psm1
 ```
 
-The resource is now discoverable by using the Get-DscResource cmdlet, and its properties are discoverable by either that cmdlet or by using **Ctrl+Space** auto-complete in the Windows PowerShell ISE.
+The resource is now discoverable by using the `Get-DscResource` cmdlet, and its properties are
+discoverable by either that cmdlet or by using **Ctrl+Space** autocomplete in the Windows
```

Comment:
Change keypress sequences to the following markup:

```
<kbd>Ctrl</kbd>+<kbd>Space</kbd
```

Reply:
done